src/Rebing/GraphQL/GraphQL.php

```diff
@@ -118,14 +138,14 @@ public function queryAndReturnResult($query, $params = [], $opts = [])
         return $result;
     }
 
-    public function addTypes($types)
+    public function addTypes(array $types): void
     {
         foreach ($types as $name => $type) {
             $this->addType($type, is_numeric($name) ? null : $name);
         }
     }
 
-    public function addType($class, $name = null)
+    public function addType(string $class, string $name = null): void
```

Comment:
This is actually a unintended break 💥 

The code within says:
```php
        if (! $name) {
            $type = is_object($class) ? $class : app($class);
            $name = $type->name;
        }
```

So objects are supported for `$class` too.

We've no test for this but my experiments with phpstan caught this => will follow-up with a fix soon